config/locales/en.models.yml

```diff
@@ -267,3 +270,7 @@ en:
         remittance_information: Bill To
       nufs_account:
         account_number: Payment Source Number
+      training_request:
+        user: Name
+        product: Instrument
```

Comment:
All of the product types, really, though I don't know if it's possible through the UI to require approval for Bundles. There are specs around restricted Bundles (which I updated) but is that something that happens?